src/Socket.cpp

```diff
@@ -30,7 +30,7 @@
 #ifdef HAVE_LIBSSL
 // Copypasted from
 // https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28default.29
-// at 22 Dec 2014
+// at 2 Jan 2016
```

Comment:
Could it be possible to change this to the ISO 8601 format (2016-01-02)? I know the previous one wasn't, but I think it would be positive change and comments would be welcome to it.